### PR TITLE
[exiv2] update to 0.28.1 and fix build error

### LINKED ISSUE #35000: Exiv2 update to 0.28.1
### Library name

Exiv2

### New version number

0.28.1

### Other information that may be useful (release notes, etc...)

Exiv2 shared library is now published to 0.28.1. Current one in VCPKG is 0.28.0

Also, the video metadata support in Exiv2 must be disabled. The Cmake option at configuration time must always be turned OFF. Video metadata code in Exiv2 is full of bugs. This feature is deprecated and will be removed in the future.

Best regards

Gilles Caulier

### PR DESCRIPTION
**Pull request by JonLiu1993**
Fixes #35000

1. Update port exiv2 to the latest version 0.28.1
2. Fix build error by add `target_link_libraries(exiv2-xmp PRIVATE expat::expat)`
3. Remove  `video` feature because video metadata code in Exiv2 is full of bugs. This feature is deprecated and will be removed in the future.

All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

Test usage successfully by `Exiv2[*]:x64-windows`:
````
Total install time: 1.5 min
exiv2 provides CMake targets:

    find_package(exiv2 CONFIG REQUIRED)
    target_link_libraries(main PRIVATE exiv2lib)
````

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/exiv2/portfile.cmake [hunk 1]
```diff
@@ -39,14 +37,15 @@ vcpkg_cmake_configure(
         -DEXIV2_ENABLE_DYNAMIC_RUNTIME=${EXIV2_CRT_DYNAMIC}
         -DEXIV2_ENABLE_WEBREADY=OFF
         -DEXIV2_ENABLE_CURL=OFF
+        -DEXIV2_ENABLE_VIDEO=ON
```
Comment (data-queue) [inline ports/exiv2/portfile.cmake:40]: Should this option be `OFF`?

File: ports/exiv2/vcpkg.json [hunk 2]
```diff
@@ -48,9 +47,6 @@
         "zlib"
       ]
     },
-    "video": {
```
Comment (data-queue) [inline ports/exiv2/vcpkg.json:51]: I don't think we should remove a feature if it is still supported upstream. We can add a comment in the description saying that it is deprecated.

File: ports/exiv2/portfile.cmake (hunk 1 above)
Comment (JonLiu1993) [reply ports/exiv2/portfile.cmake:40]: Oh, thank you for reminding me, it was a stupid mistake.

File: ports/exiv2/vcpkg.json (hunk 2 above)
Comment (JonLiu1993) [reply ports/exiv2/vcpkg.json:51]: Done

**Pull request merged**